### ISSUE TITLE
Need new release

### ISSUE THREAD

**Issue by timbrinded** [polkadot-js/api#5704]
## **I'm submitting a ...**

  - [ ] Bug report
  - [X] Feature request
  - [ ] Support request
  - [ ] Other

## **What is the current behavior and expected behavior?**

There are some features that are merged into master which I need available in my projects. We haven't had a new version in 2 months so would be worth having a new release to give access to these features.

## **What is the motivation for changing the behavior?**
Need access to the new EthereumRPCRuntimeApi types so i can generate a new typescript api-augment for moonbeam.

## **Please tell us about your environment:**

  - Version:
  - Environment:

    - [x] Node.js
    - [ ] Browser
    - [ ] Other (limited support for other environments)

  - Language:

    - [ ] JavaScript
    - [x] TypeScript (include tsc --version)
    - [ ] Other

**Comment by jacogr** [comment]
I'm acutely aware of the lack of releases - previously we have been on a weekly cycle, so it is a major knock. Sadly I've been dealing with a metric-shit-ton of issues relating to health/hospitals/etc, so the time has not been there to spend on any of the repos in a concentrated fashion.

TL;DR Aware of it, things are moving at glacial speed everywhere - there seems to be some light since some repos have received releases in the last couple of days, doesn't help here, but at least _some_ movement somewhere...

**Comment by timbrinded** [comment]
Done!
